carla/recourse_methods/catalog/actionable_recourse/model.py

```diff
@@ -223,12 +224,12 @@ def get_counterfactuals(self, factuals: pd.DataFrame) -> pd.DataFrame:
             try:
                 fs_pop = fs.populate(total_items=self._fs_size)
             except (ValueError, KeyError):
-                print(
+                log.info(
```

Comment:
That could potentially also be a `log.warning()`? Seems like something unexpected?

Reply:
Makes sense 👍